smartsim/_core/generation/generator.py

```diff
@@ -154,89 +177,82 @@ def _gen_exp_dir(self) -> None:
             dt_string = datetime.now().strftime("%d/%m/%Y %H:%M:%S")
             log_file.write(f"Generation start date and time: {dt_string}\n")
 
-    def _gen_feature_store_dir(self, feature_store_list: t.List[FeatureStore]) -> None:
-        """Create the directory that will hold the error, output and
-           configuration files for the feature store.
+        # Prevent access to type FeatureStore entities
+        if isinstance(self.job.entity, Application):
+            # Perform file system operations on attached files
+            self._build_operations()
```

Comment:
I have this here because in the future I do not want the file to call FeatureStore.files, when this DNE -> so if a user runs a job where the entity is anything but App, it doesnt enter in

Reply:
resolving bc I removed isinstance